extractor/src/main/java/org/schabi/newpipe/extractor/kiosk/KioskList.java

```diff
@@ -80,8 +81,8 @@ public KioskExtractor getDefaultKioskExtractor(final Page nextPage,
         } else {
             if (!kioskList.isEmpty()) {
                 // if not set get any entry
-                final Object[] keySet = kioskList.keySet().toArray();
-                return getExtractorById(keySet[0].toString(), nextPage, localization);
+                final String[] keySet = kioskList.keySet().toArray(new String[0]);
+                return getExtractorById(keySet[0], nextPage, localization);
```

Comment:
I think this can be improved, in order to avoid uselessly converting the list to an array (my code was not checked, improve it if there are warnings)
```suggestion
                final String kioskId = kioskList.keySet().stream().findAny().get();
                return getExtractorById(kioskId, nextPage, localization);
```